client/src/views/ScorecardView.vue

```diff
@@ -74,7 +79,7 @@ export default defineComponent({
 
 <style scoped>
 .actions-to-take {
-  background-color: #E1F5FE;
+  background-color: #e1f5fe;
```

Comment:
nit: know you didn't add this but can you change this (and above if possible, not sure about the scss in js thing) to use the color constants defined in global.scss?

Reply:
done.